Directory.Build.props

```diff
@@ -1,35 +1,36 @@
 <Project>
-
   <PropertyGroup>
-    <_IsGitHub Condition="'$(GITHUB_ACTIONS)' == 'true'">true</_IsGitHub>
-    <_ContinuousIntegrationBuild Condition="'$(TF_BUILD)' == 'true' or '$(GITHUB_ACTIONS)' == 'true' or '$(GITLAB_CI)' == 'true'">true</_ContinuousIntegrationBuild>
-
     <!-- Assembly Info -->
     <Company>Ubisoft</Company>
     <Copyright>Copyright © $([System.DateTime]::UtcNow.Year) $(Company)</Copyright>
     <Description>GitLab REST API .NET Client Library</Description>
     <Product>NGitLab</Product>
+
     <!-- Compile Options -->
     <LangVersion>9.0</LangVersion>
     <Deterministic>true</Deterministic>
     <Features>strict</Features>
     <TreatWarningsAsErrors Condition="'$(Configuration)' != 'Debug'">true</TreatWarningsAsErrors>
 
     <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
+	
+	<!-- Package info -->
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>
+  
+  <ItemGroup>  
+    <None Include="$(MSBuildThisFileDirectory)\LICENSE" Pack="true" PackagePath="" Visible="false" />
+    <None Include="$(MSBuildThisFileDirectory)\README.md" Pack="true" PackagePath="" />
+	
+    <SourceRoot Include="$(MSBuildThisFileDirectory)/" />
 
-  <!-- SourceLink: https://www.meziantou.net/how-to-debug-nuget-packages-using-sourcelink.htm -->
-  <PropertyGroup Condition="'$(_IsGitHub)' == 'true'">
-    <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
-    <IncludeSymbols>true</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
-    <ContinuousIntegrationBuild>$(_ContinuousIntegrationBuild)</ContinuousIntegrationBuild>
-  </PropertyGroup>
+	<PackageReference Include="MinVer" PrivateAssets="All" Version="2.5.0" />
 
-  <ItemGroup Condition="'$(_IsGitHub)' == 'true'">
-    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0-*" PrivateAssets="All"/>
```

Comment:
Why do we no longer use `Microsoft.SourceLink.GitHub`?

Reply:
The package `DotNet.ReproducibleBuilds` includes everything, including a dependency on `Microsoft.SourceLink.GitHub` :)

You can check the nuspec file in https://github.com/dotnet/reproducible-builds/tree/main/src/DotNet.ReproducibleBuilds